packages/react/src/auto/shadcn/inputs/relationships/RelatedModelOption.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import type { Option } from "../../../interfaces/AutoRelationshipInputProps.js";
+import type { ShadcnElements } from "../../elements.js";
+import { makeShadcnListMessages } from "./ShadcnListMessages.js";
+
+// TODO: This is Polaris's AutoSelection enum, but it might not be needed for shadcn.
+export declare enum AutoSelection {
```

Comment:
I am into this prop, but, is it actually used in the implementation? I can't see it being used so if not I think we should :fire: and if so, is the `declare` keyword actually needed here?

```suggestion
export enum AutoSelection {
```